lib/msal-browser/src/interaction_handler/RedirectHandler.ts

```diff
@@ -116,6 +116,12 @@ export class RedirectHandler extends InteractionHandler {
         authCodeResponse.nonce = cachedNonce || undefined;
         authCodeResponse.state = requestState;
 
+        // Add CCS parameters if available
+        const cachedCcsCred = this.browserStorage.getTemporaryCache(TemporaryCacheKeys.CCS_CREDENTIAL, true);
+        if (cachedCcsCred) {
+            this.authCodeRequest.ccsCredential = JSON.parse(cachedCcsCred) as CcsCredential;
```

Comment:
Same question. Should these few lines be factored into a helper function?

Reply:
+1

Reply:
added a helper with a try/catch log